virtual-setup/windows-instructions.md

```diff
@@ -1,7 +1,78 @@
-## Windows set up instructions for virtual workshops
+# Windows set up instructions for virtual workshops
 
-TOC
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-### Zoom
+- [Windows set up instructions for virtual workshops](#windows-set-up-instructions-for-virtual-workshops)
+  - [Zoom Installation and Setup](#zoom-installation-and-setup)
+    - [New Installation](#new-installation)
+    - [Set Up Preferences](#set-up-preferences)
+  - [Slack](#slack)
 
-### Slack
\ No newline at end of file
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Zoom Installation and Setup
```

Comment:
I cahnged mine back to just `## Zoom`, so links from other pages are consistent.